docs/contributing/documentation.md

```diff
@@ -190,16 +194,38 @@ There are some special cases that appear within the Enact framework. One example
 * `@param` should be repeated for each parameter. The type(s) is set within curly braces, followed by the name of the parameter. A short description of the parameter follows. Default parameter values are documented by including them within square brackets in the name position as shown above.
 * `@returns` indicates the type of the return. Functions that have no return should be documented as returning `{undefined}` to keep ESLint happy.
 
+## The Enact Voice
+
+Enact docs should strive to have a common voice.  They should be concise, informative and, as appropriate, a bit playful.  Don't be afraid to inject a little bit of fun into what could otherwise be dry reading.
+
+Tips:
+
+* Use the active voice when writing docs.
+* Descriptions should focus on what the componet or property provides.
+* Only document the `true` state for boolean propertues unless the `false` state's operation is unclear.
+
+Good:
+
+* A Moonstone-styled button with built-in support for ...
+* The color of the underline beneath button's content.
+* When `true`, the control is in the expanded state with the contents visible.
```

Comment:
Your example doesn't include the words from your template. Based on the template it would read:

> The `open` prop opens the ExpandableItem to reveal its contents

I am not a fan of that template as I feel it's even less meaningful than "When true...". That being said, I do think the wording of the example is acceptable:

> Opens the `ExpandableItem` to reveal its contents

It describes the action as a sentence fragment stating by implying the relevant property name.

Reply:
As commented below, 'Opens the...' implies some sort of imperative operation.  Which is why I had used the the 'when true,  ....' because it tells you the result when it's true.  It doesn't imply any sort of transition or action.  I am open to better wording, of course.  I was trying to avoid things like 'Controls the open status of the ExpandableItem', though it as descriptive.

Reply:
@Djspaceg I meant that YYY would the text used for the description such that it would read like the template. Sorry I wasn't clear on that.

To close the loop on the two comment threads, should props that "do" things (e.g. `open`) begin with verbs (e.g. "Opens ...") and props that "are" things (e.g. `backgroundColor`) not being with verbs (e.g. "The color used in the back ..." ;) )?

In a declarative framework, changing a prop is the equivalent of invoking a method so I think the use of verbs can be appropriate in some cases.

Reply:
I don't entirely agree.  If I put this in my code:

```
<Popup open />
```

It does not render a popup and open it.  It renders an open popup.  Maybe the distinction is subtle (and, maybe, overstated?) but I think that saying props invoke methods is not accurate in many, many circumstances.